### ISSUE TITLE
Entropy/Life Drain Overpowered?

### ISSUE THREAD

**Issue by jjnich** [openlegend/core-rules#79]
It seems like the Life Drain bane is overpowered making Entropy a pretty powerful attribute. Basically every attack you do with Entropy you can gain the damage you deal back as hit points. I think it should be more like the Heal boon where depending on how much damage you do you roll again for what you heal? or you get half the damage you deal back? or something else to make it just a little bit less powerful, maybe add a line in Life Drain that says "This boon can only target one enemy"?

idk just something to bring it down a bit.

**Comment by raihje** [comment]
There's a bit of a trade off. For heal, you roll to invoke and can heal yourself or others. With Life Drain, you need to target a creature's defense (potentially easier/harder) and that creature needs to have hitpoints for you to steal. In other words, in a drawn out fight a character spamming life drain has a huge advantage when a big HP creature or several small HP creatures are present. But in small skrimishes or against high-def enemies heal is a bit more reliable - also, Regenerate is pretty solid for drawn out fights since it's one roll to invoke and then sustain it. Life Drain is a one-and done similar to heal, but targeting an enemies defense ( and doing damage)

Basically, Life Drain isn't terribly overpowered, just powerful in a different way, if you're fighting creatures comparative to your level Heal/Regenerate are on par and more reliable. Life Drain would mop the floor with anything that has a low defense though making it vastly superior in your classic "low level party vs goblins/mooks"

**Comment by jjnich** [comment]
So if you do  Entropy and Creation you can have Regeneration going from the first round and life drain every round of combat.
I'm just saying in my game a player went with 5 Entropy and was very powerful.

**Comment by jjnich** [comment]
also is there anything stopping you from having Regeneration going constantly? Roll once to invoke (outside of combat) and just keep sustaining . . . What happens if you fail to invoke a boon outside of combat?

**Comment by raihje** [comment]
Outside of combat it's up to the GM. Keep in mind all actions are available to the enemy, if they see your wounds closing  they might be smart enough to do a Disrupting attack or something.

**Comment by jjnich** [comment]
Yeah, I'm the GM, that's why I'm asking what happens when you fail to invoke a boon outside of combat. Like I get it most of the time, you fail to pick a lock, you break your picks but still get through, or you fail a sneak, someone notices you and enemies appear. I'm trying to come up with a good example of "I failed to invoke my levitate boon outside of combat so XXXXXXXX happens" I'm wondering if you can come up with an actually good response to "fail but the story progresses" when attempting to invoke a boon outside of combat (or really even in combat), rather than just saying "you fail to fly/teleport/heal/whatever"

I'm trying to say this will be one particularly difficult spot to apply the core mechanic, so I think there should at least be a couple examples in the core rules of what to do when failing an invocation of a boon (both in and out of combat).

**Comment by raihje** [comment]
You could end up manifesting your powers in an unintended way, or you burn a significant amount of time.

**Comment by brianfeister** [comment]
Outside of combat, I dilate time and prevent characters from invoking their abilities once every 6 seconds for the sake of keeping the game interesting.

So, it's worth noting that often times GMs should simply enable "auto-success" outside of combat and not even bother rolling.

If a character has regeneration (consider that all characters heal to full HP after 10 minutes of rest), then I would probably have them heal to full in 5 minutes with a successful Regeneration invocation check. So the question of Regeneration over the course of 3 hours doesn't matter much since everyone heals in that time. 

A better question and example would be `Invisibility` -- if I'm invisible, how long can I sustain it? Well, the answer is "As long as you have an available minor action to sustain the boon".  

If I were the GM and you had a character invoking `Invisible` outside of combat, I would have you make a single roll. Let's say you're trying to ambush someone. On a really bad failure, I would say that you're hiding behind a crate and the rough cut wood catches and rips some of your clothing just as your target is approaching. This causes you to lose concentration and become visible.

If your roll was close to succeeding, I might say that your clothes get caught but DO NOT rip / make a sound, so you are momentarily inconvenienced and you have to spend your first Major action freeing yourself, but you remain invisible and can act normally the next round.

**Comment by raihje** [comment]
As for Life Drain, I don't have much experience with that attribute in game (or a lot of experience with the game period, nobody seems to be running online with spots open lol) But I think the tradeoff @brianfeister is going for is you need to hit an enemy (which could possible defend) vs a reliable X amount healed. Life Drain is really powerful though against anything with a comparative defense - this situation alone might be reason to nerf it a bit as otherwise the DM is limited to very high toughness mobs if he wants to present as much of a challenge to the party as he feels necessary/wants to in the story. If I were to say something it'd be half damage healed, that way if you get a solid hit you're healing good, but you're not the unending representation of death. Half Damage would be fairly reasonable and easy to do on the fly.

**Comment by brianfeister** [comment]
@jjnich @raihje - I didn't really respond because I've been mulling it over, but I think you're right that `Life Drain` is probably OP. I think the simplest fix might be to have you heal 50% of the damage dealt, rounded up. That means Bane Focus (Life Drain) would just yield 3 HP of healing. 

Also, I think we need a **Special** tag on this bane to note that `Special: this bane cannot be used as part of a Bane Attack, only a Damaging Attack`.  I'm not sure, thinking on that one...

@istabosz , what are your thoughts on this? I'm really hesitant to have a bane that doesn't work for a standard Bane Attack, but the damage hinges on the Action Roll, and if we allow the Advantage X from Bane Focus to stack on top of this, I'm pretty sure it's OP

**Comment by istabosz** [comment]
What if we just turn life drain into a boon? While it's active, you heal 50% of the damage you dish out.

**Comment by brianfeister** [comment]
That sounds PERFECT

**Comment by jjnich** [comment]
I agree, that's great. Then you can't do life drain and regeneration at the same time. How will it effect multiple targets though? If I put it on 3 allies do they all heal when I do damage or each person heals when they do damage? I'm assuming the later as that makes more sense and would be easier to do I think.

**Comment by raihje** [comment]
Maybe make it so it's on 3 enemies, self-target only. That way if you take the disadvantage (or specialize) in area attacks or multi-targeting then you have the opportunity to heal more.

**Comment by istabosz** [comment]
I've added life drain as a boon and removed it as a bane.

The text makes it clear that the target of the boon heals when he deals damage. So if I cast it on Ally the Elf, Ally heals when he attacks, not when I do.
